server/lib/users/user-model.ts

```diff
@@ -341,22 +341,29 @@ export async function findUserByName(name: string): Promise<SbUser | undefined>
 }
 
 /**
- * Returns a `Map` of name -> `User` for a given list of names. Any names that can't be found won't
- * be present in the resulting `Map`.
+ * Returns the data for all users with the specified names. If a user cannot be found it will not
+ * be included in the result. The order of the result is not guaranteed.
  */
-export async function findUsersByName(names: string[]): Promise<Map<string, SbUser>> {
-  // TODO(tec27): Should this just return an array and let callers make the Map if they want?
+export async function findUsersByName(names: string[]): Promise<SbUser[]> {
```

Comment:
In the future I would vastly, vastly, vastly prefer that you make this change in a separate PR and rebase your code on top of it, doing this here makes all of this much harder to review